lld/test/ELF/lto/arm-rtlibcall.ll

```diff
@@ -0,0 +1,125 @@
+; REQUIRES: arm
+;; https://github.com/llvm/llvm-project/issues/127284
+;; Test for LTO optimizing out references to symbols that are pulled in by
```

Comment:
"optimizing out references " might not be the best way to phrase this issue. As this is no "optimizing out" step involved. My phrasing in https://github.com/llvm/llvm-project/issues/127284#issuecomment-2661091430 might be useful?